LayoutTests/platform/gtk/TestExpectations

```diff
@@ -849,21 +849,18 @@ fast/forms/range/input-appearance-range-rtl.html [ ImageOnlyFailure ]
 
 # Service-workers tests that fail, time out or crash.
 
-# GTK ought to support async clipboard
-webkit.org/b/211979 editing/async-clipboard/clipboard-change-data-while-writing.html [ Failure ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-read-text-from-platform.html [ Failure ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-change-data-while-getting-type.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-change-data-while-reading.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-get-type-with-old-items.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-read-basic.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-read-text-same-origin.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-write-basic.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-write-items-twice.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-write-text.html [ Timeout ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-item-get-type-basic.html [ Failure ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-read-text.html [ Failure ]
-webkit.org/b/211979 editing/async-clipboard/clipboard-read-write-images.html [ Failure ]
-webkit.org/b/211979 editing/async-clipboard/sanitize-when-reading-markup.html [ Failure ]
+# GTK doesn't support multiple items in the clipboard
+# FIXME: add platform specific tests equivalent to these but using single item.
+editing/async-clipboard/clipboard-read-write-images.html [ Skip ]
+editing/async-clipboard/clipboard-write-basic.html [ Skip ]
+editing/async-clipboard/sanitize-when-reading-markup.html [ Skip ]
+
+# These fail because are run with DOMPasteAllowed=false or JavaScriptCanAccessClipboard=false,
+# I don't know how they are supposed to work.
```

Comment:
Nit - grammar ("These fail because are run").

These layout tests work by triggering programmatic paste (i.e. the page calls `document.execCommand("paste")` underneath user activation). On both macOS and iOS, this causes a contextual menu to be shown that gives the user an option to paste (thereby allowing the page to read from the system pasteboard). Since they're intended to exercise this programmatic paste mechanism, we explicitly turn `DOMPasteAllowed` off.

It sounds like we could make this work in GTK as a followup, but I think either skipping or marking these as failures/timeouts is the right call for now.

Reply:
Thanks for the clarification, I've changed the comment and updated the expectations to skip to avoid timeouts.